docs/assets/C#NamingGuidelinesTable.md

```diff
@@ -0,0 +1,15 @@
+# C# Naming Conventions - Quick Reference Table
+
+| Kind                                 | Naming Convention |  Example                                                           |
+| ------------------------------------ | ----------------- | ------------------------------------------------------------------ |
+| Classes                              | PascalCase        | `class Car{}`                                                      |
```

Comment:
Not sure the curly braces are adding much, especially since they are omitted below on things like method.
```suggestion
| Classes                              | PascalCase        | `class Car`                                                      |
```

Reply:
Yeah, I was struggling to determine how much information should be in this table vs just in the guidelines; @MarkMichaelis  also proposed an additional explanation column but I thought it would start to bloat a table that has the purpose of being quick reference, not to go into depth of explanations, as that is what the guidelines themselves are for.